README.md

```diff
@@ -1,28 +1,36 @@
 # Freifunk Berlin Firmware
 https://wiki.freifunk.net/Berlin:Firmware
 
-This is the build-system for the Firmware of Freifunk Berlin.
-The firmware is based on vanilla [OpenWrt](https://openwrt.org/start) with some modifications (to fix
-broken stuff in OpenWrt itself or for example LuCI) and additional default packages/configuration settings.
+This is the build-system for the Firmware of Freifunk Berlin. The firmware is
+based on vanilla [OpenWrt](https://openwrt.org/start) with some modifications
+(to fix broken stuff in OpenWrt itself or for example LuCI) and additional
+default packages/configuration settings.
+
 
 ## Contact / More information
 
-More user relevant information about the firmware are on the wiki page at: https://wiki.freifunk.net/Berlin:Firmware. There you can also find the
-* [ReleaseNotes](https://wiki.freifunk.net/Berlin:Firmware/v1.0.2)
-* a tutorial ([en](https://wiki.freifunk.net/Berlin:Firmware:En:Howto) / [de](https://wiki.freifunk.net/Berlin:Firmware:Howto)) on router configuration
+More user relevant information about the firmware are on the wiki page at: https://wiki.freifunk.net/Berlin:Firmware.
+There you can also find the
+* [ReleaseNotes](https://wiki.freifunk.net/Berlin:Firmware/v1.0.3)
+* a tutorial ([en](https://wiki.freifunk.net/Berlin:Firmware/en/HowTo) / [de](https://wiki.freifunk.net/Berlin:Firmware/HowTo)) on router configuration
+
+For questions write a mail to <berlin@berlin.freifunk.net> or come to our
+weekly meetings. If you find bugs please report them at: https://github.com/freifunk-berlin/firmware/issues
 
-For questions write a mail to <berlin@berlin.freifunk.net> or come to our weekly meetings.
-If you find bugs please report them at: https://github.com/freifunk-berlin/firmware/issues
 
 ## Development
 
 ### Info
 
-For the Berlin Freifunk firmware we use vanilla OpenWrt with additional patches
-and packages. The Makefile automates firmware
-creation and apply patches / integrates custom freifunk packages. All custom
-patches are located in *patches/* and all additional packages can be found at
-http://github.com/freifunk-berlin/packages_berlin.
+How does this stuff work? 
+
+The idea is to download OpenWrt via git, patch it with all patches in the
+`patches/` folder and configure it according to the configuration snippets in
+`configs`.  Then use the OpenWrt buildsystem to build all packages and the
+Imagebuilder and use the latter to assemble the final image with lists of
+packages in packages for the router.
+All additional packages can be found at http://github.com/freifunk-berlin/packages_berlin.
```

Comment:
should we mention "gluon-packages" and "freifunk-potsdam-packages" also. (not shure if ffp-packages made it up to this repo already)